src/server/memory_cmd.cc

```diff
@@ -338,7 +341,7 @@ void MemoryCmd::Track(CmdArgList args) {
       return cntx_->SendError(parser.Error()->MakeReply());
     }
 
-    atomic_bool error;
```

Comment:
was it wrong before? 🤔

Reply:
https://stackoverflow.com/questions/36320008/whats-the-default-value-for-a-stdatomic